packages/boilerplates/Boilerplate.ts

```diff
@@ -0,0 +1,75 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { IDependency } from '@dolittle/tooling.common.dependencies';
+import { BaseBoilerplate, Scripts } from './internal';
+
+export const boilerplateContentFolderName = 'Content';
+/**
+ * Represents the standard boilerplate, meaning a non-artifacts boilerplate 
+ */
+export class Boilerplate extends BaseBoilerplate {
```

Comment:
This looks a bit weird. I'm expecting Boilerplate to be the base abstract representation of a boilerplate, while there is another layer here with a BaseBoilerplate - this suggests to me that the naming is not specific enough.

Reply:
Correct, it's not. Naming stuff is hard :)